docs/contributing/use-reliable-links.md

```diff
@@ -0,0 +1,104 @@
+## Use reliable links
+
+If you would like to link to a website,
+do your best to link to a reliable link destination.
+
+Even if you update the link in the repo if it changes or breaks,
+outdated versions might remain in older versions of tools, docs, etc.
+Sometimes the link destination itself can add an appropriate redirect,
+but sometimes it can't and other times it's destination is not the best choice.
+
+- If you're not sure if you need a more reliable link,
+  check out [Situations to consider](#situations-to-consider).
+- If your destination does not already have a stable link,
+  and you think it would benefit from one,
+  follow the instructions in [Create a reliable link](#create-a-reliable-link).
+
+### Situations to consider
```

Comment:
Are there any situations where we shouldn't use a reliable link? I am wondering if we should have a check that disallows any non-reliable links from getting submitted.

Reply:
I mean I think you should always use as reliable of a link as possible, but I'm hesitant to prescribe that every link should use this format as there is still a maintenance cost. Perhaps for doc links output from the `flutter` tool or error messages.

I'd like to try this more informal, intentional process first and then after two releases or so, revisit some surrounding tooling or checks. How does that sound?

Reply:
Sounds good.